setup.py

```diff
@@ -24,7 +24,7 @@
 
 setup(
     author="OpenCDMS",
-    author_email="info@opencdms.org",
+    author_email="info@opencdms_process.org",
```

Comment:
Whoops! (this should remain as info@opencdms.org)

Reply:
ohh.. fixing it right away!

Reply:
https://github.com/opencdms/opencdms-process/blob/f9861474123cb052389a859f649bc466673fa6eb/opencdms_process/__init__.py